Automattic/jetpack
# Edit User: hide profile fields for WPCOM users.

Disclaimer: I am not so happy with the workaround I used here, please advise if there is a better way!
Info: `personal_options` is triggered right before the fields are rendered.

Relates to https://github.com/Automattic/wp-calypso/issues/46077

#### Changes proposed in this Pull Request:

Hides profile fields ( First Name, Last Name, Display Name, About, Email, Website ) for WordPress.com users. These fields can be edited in WordPress.com from their respective users.


#### Jetpack product discussion


pd2C0z-1T-p2

#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Load this branch on Jetpack Beta of your Atomic site

Scenario 1 - Current User
- Go to `/wp-admin/profile.php`
- Make sure that fields ( First Name, Last Name, Display Name, About, Email, Website ) are hidden and a WordPress.com link is displayed

Scenario 2 - Another Connected User
- Go to `/wp-admin/users.php` and select another connected user
- Make sure that fields ( First Name, Last Name, Display Name, About, Email, Website ) are hidden and a WordPress.com link is displayed

Scenario 2 - Another local user
- Go to `/wp-admin/users.php` and select a local user
- Make sure that fields are not hidden.


Before | After
-------|------
![](https://cln.sh/tcfwp6+) | ![](https://cln.sh/HL2ley+)
